### DISCUSSION TITLE
Beatmap update button restarts animation at every hover

### DISCUSSION THREAD

**Discussion by cw-urn** [ppy/osu#31123] [Ideas] [answered]
Currently, when hovering over an "Update" button, inside the song select menu, the spinning circle animation replays each time.

![gif](https://github.com/user-attachments/assets/54654540-a355-4222-a2aa-a00ca847b346)

Similar to other UI elements, the spinner should reset with an ease after the hovering ends.

**Comment by peppy** [comment]
Not against this if someone wants to give it a try.

**Reply by peppy** [reply, marked as answer]
Fixed in https://github.com/ppy/osu/pull/34283.
